### ISSUE TITLE
About a question of test_training_cifar.py

### ISSUE THREAD

**Issue by lastonephy** [Vanint/TADE-AgnosticLT#13]
In line 200 and 201 of test_training_cifar.py:
`           dataset = IMBALANCECIFAR100(data_dir, train=True, download=True, transform=train_trsfm, imb_type=imb_type, imb_factor=test_imb_factor, reverse=reverse) 
       `
`train_dataset = IMBALANCECIFAR100(data_dir, train=True, download=True, transform= TwoCropsTransform(train_trsfm), imb_type=imb_type, imb_factor=test_imb_factor, reverse=reverse) `
why you set the **train** is **True**? I think it should be **False** to obtain the weighting parameters of test set. 
Can you explain it?
Thanks!

**Comment by YifanZhang07** [comment]
Hi. thanks for mentioning this.
We double-check the test_training_cifar.py and find that this file is the wrong version when we clear up the open-source file.
In fact, **you will obtain the results reported in our paper or on GitHub by changing Line 201 to** 
```
train_dataset = IMBALANCECIFAR100(data_dir, train=False, download=True, transform= TwoCropsTransform(train_trsfm), imb_type=imb_type, imb_factor=test_imb_factor, reverse=reverse)
```

We have modified the corresponding code file on Github. Thanks again.

**Comment by YifanZhang07** [comment]
If you have any further questions, please let me know~
